src/lib/dev-environment/dev-environment-lando.js

```diff
@@ -294,3 +294,17 @@ async function ensureNoOrphantProxyContainer( lando ) {
 
 	await proxyContainer.remove();
 }
+
+export async function ensureDockerInstalled() {
+	const lando = new Lando( getLandoConfig() );
+	await lando.bootstrap();
+
+	lando.log.verbose( 'docker-engine exists: %s', lando.engine.dockerInstalled );
+	if ( lando.engine.dockerInstalled === false ) {
+		throw Error( 'docker could not be located!' );
```

Comment:
TIL that `throw Error()` and `throw new Error()` [are equivalent](https://tc39.es/ecma262/#sec-error-constructor) 😀

Reply:
What do you think about checking for both first and then throwing a combined error instead? 

Also providing links to Docker site may be a nice touch and spare extra click.

As for the error message if on mac or windows "Please install Docker Desktop", if on Linux: Please install Docker Engine and Docker Compose.

Reply:
Good idea with the docs. I added those.

I was trying to avoid doing platform checks there. So I opted to not do different text per platform if you don't feel strongly about that?

Same for having the message composed as the scenario when you have docker but not docker-compose is kinda rare (linux only), and it seems not for long as they are rolling out Docker Desktop to linux :(

Reply:
> I was trying to avoid doing platform checks there. So I opted to not do different text per platform if you don't feel strongly about that?
Makes sense!